spec/requests/shared_examples/api/v4/shared_spec_for_sync_requests.rb

```diff
@@ -158,4 +158,32 @@ def assert_sync_success(response, request_process_token)
       expect(JSON(response.body)[response_key].count).to eq(1)
     end
   end
+
+  context "translations are correctly inferred" do
```

Comment:
These tests are coupled to our application config currently, it would be nice to decouple the unit test from the application config. One option could be to mock the application config for these tests.

Reply:
Mocking application config would defeat the purpose of these tests, which IMO, can only be done well using integration tests.